compiler.go

```diff
@@ -10,7 +10,7 @@ import (
 var (
 	DefaultCompiler PatchCompiler = &defaultCompiler{}
 
-	BulkCompiler PatchCompiler = &bulkCompiler{}
+	BulkCompiler PatchCompiler = &BulkCompilerWithOption{MaxPatches: 1000}
```

Comment:
default value should `0` (unlimited) .
the placeholders limit is 999 when use sqlite3 driver.

if we have to use this option should create new instance with flavored options.

Reply:
fixed